### DISCUSSION TITLE
blog/photo/2024/202408

### DISCUSSION THREAD

**Discussion by giscus** [yyyytawa/yyyytawa.github.io#13] [Announcements]
# blog/photo/2024/202408

butlanys发的管我dream啥事 PS: butlanys其实就是dream,只是后面改名了 butlanys_and_dream 请点击修改第II卷的文字说明 为啥点了没反应啊,在线求助 click_to_modify

https://docs.yyyyt.top/blog/photo/2024/202408.html

**Comment by runoneall** [comment]
鲁迅说的关我周树人什么事
